Makefile

```diff
@@ -122,9 +122,9 @@ CUOBJ = $(patsubst %.cu, build/%_gpu.o, $(CUSRC))
 
 # extra operators
 ifneq ($(EXTRA_OPERATORS),)
-	EXTRA_SRC = $(wildcard $(patsubst %, %/*.cc %/*/*.cc, $(EXTRA_OPERATORS)))
+	EXTRA_SRC = $(wildcard $(foreach DIR, $(EXTRA_OPERATORS), $(foreach PAT, *.cc */*.cc, $(DIR)/$(PAT))))
```

Comment:
patsubst won't replace multiple patterns. 

```
# i.e.
$(patsubst %, %/*.cc %/*/*.cc, $(EXTRA_OPERATORS) will end up with:
$(EXTRA_OPERATORS)/.cc %/*/*.cc
```
At least not on ubuntu 14.04, can you confirm that?